libraryBackend/libraryApp/models.py

```diff
@@ -153,3 +153,66 @@ class Librarian(User):
 
     def __str__(self):
         return self.user.email
+
+
+class Book(models.Model):
+    isbn = models.CharField(max_length=13)
+    name = models.CharField(max_length=30)
+    publisher = models.CharField(max_length=30)
+    author = models.CharField(max_length=30)
+    publish_year = models.CharField(max_length=4)
+    quantity = models.IntegerField()
+    demand = models.BigIntegerField()
+    is_available = models.BooleanField(default=True)
+
+    def __str__(self):
+        return self.name + "_" + self.author
+
+
+class Copy(models.Model):
+    book = models.ForeignKey(Book, on_delete=models.CASCADE)
+    condition = models.CharField(max_length=30)
```

Comment:
Make this a choice field. And you can assign a fine value of each condition. That way the system could automatically fine depending on the condition